delphi/app/graph.py

```diff
@@ -1,75 +1,233 @@
-"""
-Contains classes and utilities for graph operations.
-"""
 
-class GraphNode():
+def find_path(start_node, flow, people):
     """
-    Contains information about each node within the graph representation
+    find_path is a utility function that returns all that nodes that are able to carry flow from a specific source
+    node
+    :param start_node: The source node to calculate paths for
+    :param flow: List of all nodes in the network with positive flow
+    :param people: The amount of people that must be carried from the source node
+    :return: A list of all possible nodes that contribute to viable paths for start_node
     """
+    path_list = [start_node]
+    for node in path_list:
+        current_node = node
+        for edge_check in flow:
+            if edge_check.start == current_node and edge_check.end != 'ta':
+                if edge_check.flow >= people / 2:
```

Comment:
what does the people/2 represent?

Reply:
This is getting deleted out of graph.py. It's going to now be located in the plugin